### PR TITLE
Add total earned

### PR DESCRIPTION
**Pull request by bcolombini**
Add total earned by currency, and creating function in poloniex class.

### CODE REVIEW

**Comment by lalbuquerque** [comment]
Very useful!

**Comment by rnevet** [comment]
Need to revise on master.

**Comment by bcolombini** [comment]
Ok, i found a problem with poloniex API. This api return only 188 last loan but i  when found solution i will make another pull to your review. Thanks

Bruno Colombini''
(21)9967-0004
Developer para iphone e sistemas Web.
----------------------------------------
bruno.colombini@me.com

**Comment by prfiredragon** [comment]
@bcolombini , I make some tests. I think that I found something that may help you. but you most test it.
You need to create an entry in the configuration for the initial date of trade or from when you want to display the data. for the start variable.

on lendingbot.py
`	ts = time.time()
	end = datetime.datetime.fromtimestamp(ts).strftime('%d/%m/%Y')
	start = '01/07/2015' 
	timestampStart = time.mktime(datetime.datetime.strptime(start, "%d/%m/%Y").timetuple())
	timestampEnd = time.mktime(datetime.datetime.strptime(end, "%d/%m/%Y").timetuple())
	earned = earnedTotal(bot.returnLendingHistory(timestampStart,timestampEnd))`

on poloniex.py
`
    def returnLendingHistory(self,start,stop):
        return self.api_query('returnLendingHistory',{'start':start,'end':stop})`

**Comment by praeluceo** [comment]
This functionality would be extremely useful to the bot, especially if it could list lifetime earnings by the  bot, as well as broken down by monthly revenue. I have a spreadsheet where I manually run the report by comparing the Unix timestamps between two dates ala:
[https://poloniex.com/tradeHistory#analysis/BTC/-/lending/1465516800/1465603199](url)

But to have this automatically maintained in realtime by the bot would be awesome. I'd be willing to tip for this feature!

**Comment by ckcollab** [comment]
@praeluceo I could do this, please contact me :)

For my reference: https://poloniex.com/tradeHistory#analysis/BTC/-/lending

File: lendingbot.py [hunk 1]
```diff
@@ -83,9 +82,7 @@
 parser.add_argument("-spread", "--spreadlend", help="How many orders to split your lending into")
 parser.add_argument("-gapbot", "--gapbottom", help="Percentage of your order's volume into the ledger you start lending")
 parser.add_argument("-gaptop", "--gaptop", help="Percentage of your order's volume into the ledger you stop lending")
-parser.add_argument("-60day", "--sixtydaythreshold", help="Rate at where bot will request to lend for 60 days") # only used for backward compatibility
```
Comment (ckcollab) [inline lendingbot.py]: Removing these arguments is what needs to be changed?

Comment (Evanito) [inline lendingbot.py]: This needs a complete refresh at this point.

**Comment by rnevet** [comment]
I think this needs a rewrite.
I would like to see this as a plugin "account_stats" not cluttering the main code.
It also doesn't make sense to calculate the total earned on every bot cycle, my lending history is now ~15MB.

**Comment by rnevet** [comment]
FYI, I'm working on an implementation for this with another approach.

**Pull request closed without merging**